apps/antalmanac/src/components/Calendar/toolbar/ScheduleSelect/ScheduleSelect.tsx

```diff
@@ -90,9 +90,11 @@ export function SelectSchedulePopover(props: { scheduleNames: string[] }) {
                 onClick={handleClick}
                 sx={{ minWidth, maxWidth, justifyContent: 'space-between' }}
             >
-                <Typography whiteSpace="nowrap" textOverflow="ellipsis" overflow="hidden" textTransform="none">
-                    {currentScheduleName}
-                </Typography>
+                <Tooltip title={currentScheduleName} enterDelay={200} disableInteractive placement="right">
```

Comment:
issue: tooltip should apply to the entire button, not just the text

suggestion: wrap the whole button with the tooltip